src/quagga.js

```diff
@@ -92,9 +106,9 @@ function ready(cb){
     cb();
 }
 
-function initCanvas() {
+function initCanvas(_config) {
```

Comment:
Instead of passing in the `_config` parameter, you could also use the global one (see L51)